website/src/components/LocalIdentityBadge/LocalIdentityBadge.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { IdentityBadge } from '@aragon/ui'
+import { getNetworkType } from '../../utils/web3-utils'
+
+function LocalIdentityBadge ({ entity, ...props }) {
+  return (
+    <IdentityBadge entity={entity} networkType={getNetworkType()} {...props} />
+  )
+}
+
+LocalIdentityBadge.propTypes = {
+  ...IdentityBadge.propTypes
```

Comment:
If there are no additional props I think just doing `LocalIdentityBadge.propTypes = IdentityBadge.propTypes` is fine

Reply:
Done in https://github.com/1Hive/apiary/pull/181/commits/e43aa46bf510777197f64544d278b58117c8d3f2.